### PR TITLE
feat(*): Add stop host

### PR DESCRIPTION
**Pull request by thomastaylor312**
This updates a few libraries so we could add the stop host command. I also
did a small cleanup of some clippy lints

### CODE REVIEW

File: src/ctl/mod.rs [hunk 1]
```diff
@@ -537,6 +558,18 @@ pub(crate) async fn handle_command(command: CtlCliCommand) -> Result<String> {
...
             )
         }
+        Stop(StopCommand::Host(cmd)) => {
+            let output = cmd.output;
+            sp =
+                update_spinner_message(sp, format!(" Stopping host {} ... ", cmd.host_id), &output);
+            let ack = stop_host(cmd.clone()).await?;
+            ctl_operation_output(
+                ack.accepted,
+                &format!("Host {} stopped successfully", cmd.host_id),
```
Comment (autodidaddict) [inline src/ctl/mod.rs]: Nit: I would rephrase this to the "host acknowledged stop request" .. we don't actually know if the host managed to immolate

**Review by autodidaddict — CHANGES_REQUESTED** [review]

**Review by autodidaddict — APPROVED** [review]

**Review by brooksmtownsend — APPROVED** [review]
This is gonna give me merge conflicts in #194 so I'm angry but the code looks good so I can't hold you back 😞
